konduit-serving-data/konduit-serving-nd4j/src/main/java/ai/konduit/serving/data/nd4j/format/ND4JNDArrayFactory.java

```diff
@@ -51,6 +57,14 @@ public NDArray create(Object o) {
         INDArray a;
         if(o instanceof INDArray){
             a = (INDArray)o;
+        } else if (o instanceof ListValue) {
+            List<Long> shape = new ArrayList<>();
+            List<Float> data = new ArrayList<>();
+
+            ListValue listValue = (ListValue) o;
```

Comment:
Could you explain the point of the ListValue?

Reply:
Similar to the difference between list and numpy.ndarrays in python. Other than that I don't think there's much use here

Reply:
I had to do this so that the json arrays passed as inputs have a way to be converted into INDArrays since it's inconvenient to find a object representation of INDArray and pass that in http requests

Reply:
Other than that ListValues are useful for sending nested java as well

Reply:
Also useful for sending in other data types like list of strings or list of objects